index.bs

```diff
@@ -5237,30 +5237,30 @@ partial interface MLGraphBuilder {
     : <dfn>scales</dfn>
     ::
         A list of length 2.
-        Specifies the scaling factor in each spatial dimensions of the input: *[scaleHeight, scaleWidth]*.
+        Specifies the scaling factor in each dimensions of the input in {{MLResample2dOptions/axes}} : *[scaleForFirstAxis, scaleForSecondAxis]*.
         The default value is [1.0, 1.0].
 
     : <dfn>sizes</dfn>
     ::
         A list of length 2.
-        Specifies the target sizes for each spatial dimensions of the input: *[sizeHeight, sizeWidth]*. When the target sizes are specified, the {{MLResample2dOptions/scales}} argument is ignored, since the scaling factor values are derived from the target sizes of each spatial dimension of the input.
+        Specifies the target sizes for each dimensions of the input in {{MLResample2dOptions/axes}} : *[sizeForFirstAxis, ForSecondAxis]*. When the target sizes are specified, the {{MLResample2dOptions/scales}} argument is ignored, since the scaling factor values are derived from the target sizes of the input.
```

Comment:
```suggestion
        Specifies the target sizes for each dimension of the input from {{MLResample2dOptions/axes}} : *[sizeForFirstAxis, ForSecondAxis]*. When the target sizes are specified, the {{MLResample2dOptions/scales}} argument is ignored, since the scaling factor values are derived from the target sizes of the input.
```

Reply:
Btw, I don't think we need to *explicitly* mention in documentation (because it's implicit in the definition of `*[sizeForFirstAxis, ForSecondAxis]*)` that for cases like `axes = [3, 0]` and `sizes = [640, 3]`, it means axis 3 is size 640, and axis 0 is size 3 (that is`output.dimensions[axes[0]] = sizes[0]` and `output.dimensions[axes[1]] = sizes[1]`); but it will be prudent to include a test case for it, where the axes have high axes before low axes. ⏳